flink-streaming-scala/src/main/scala/org/apache/flink/streaming/api/scala/DataStream.scala

```diff
@@ -44,7 +44,14 @@ class DataStream[T](javaStream: JavaStream[T]) {
   /**
    * Gets the underlying java DataStream object.
    */
-  def getJavaStream: JavaStream[T] = javaStream
+  private[flink] def getJavaStream: JavaStream[T] = javaStream
+
+  /**
+    * Returns the [[StreamExecutionEnvironment]] associated with the current [[DataStream]].
+    * @return associated execution environment
+    */
+  def getExecutionEnvironment: StreamExecutionEnvironment =
```

Comment:
This is consistent with other parts, but I would suggest to adopt the following style: When defining a method without parenthesis (suggesting it could transparently be a variable), it should not be called like a method. So this one should rather be called `def executionEnvironment: StreamExecutionEnvironment = ...`